packages/styles/scss/patterns/sub-patterns/content-group/_content-group.scss

```diff
@@ -37,6 +36,14 @@
 
     &__children {
       margin-bottom: $carbon--layout-03;
+
+      .#{$prefix}--image__img {
+        margin-bottom: $spacing-07;
+      }
+
+      .#{$prefix}--content-item__heading {
+        margin-top: 0;
+      }
```

Comment:
I don't think content item is apart of the content group sub-pattern.

Reply:
Yeah, didn't realize that. Will remove